README.md

```diff
@@ -0,0 +1,3 @@
+# Not a Hub
+
+Bienvenue sur le site officielle de Not a Name, une communauté francophone de programmation.
```

Comment:
Ça n'a rien à faire dans ce commit

Reply:
Si carrément :)
De base le commit c'était juste ça, puis l'action a été ajouté et du coup j'ai rajouter un and. Mais c'est ça le "site index".